components/WalletFieldList.vue

```diff
@@ -0,0 +1,185 @@
+<template>
+  <div>
+    <div
+      v-for="(items, i) in value"
+      :key="items.id"
+      class="flex flex-row items-center justify-center"
+    >
+      <button
+        :class="[
+          'h-[40px]',
+          'w-min',
+          'block',
+          'relative',
+          'px-[12px]',
+          'mr-[4px]',
+          'bg-shade-gray',
+          'rounded-[12px]',
+          'box-border',
+          'outline-none',
+          items.isOpenOptions ? 'bg-medium-gray' : 'hover:bg-light-gray',
+          'transition',
+          'duration-150',
+          'cursor-pointer',
+        ]"
+        @blur="items.isOpenOptions = false"
+      >
+        <div
+          class="
+            h-[100%]
+            w-[100%]
+            flex flex-row
+            items-center
+            justify-center
+          "
+          @click="
+            items.isOpenOptions = !items.isOpenOptions
+            isSelecting = !isSelecting"
+        >
+          <div class="w-[20px] mx-[8px]">
+            <IconLike v-if="items.type === 'cosmos'" />
+            <IconEthereum v-if="items.type === 'eth'" />
```

Comment:
Confusing that the icon for LIKE is used for cosmos, but we don't have an extra cosmos icon for cosmos

Reply:
Asking @joshkiu to provide icon for cosmos type